osu.Game/Scoring/ScoreInfoExtensions.cs

```diff
@@ -12,6 +12,6 @@ public static class ScoreInfoExtensions
         /// <summary>
         /// A user-presentable display title representing this score.
         /// </summary>
-        public static string GetDisplayTitle(this IScoreInfo scoreInfo) => $"{scoreInfo.User.Username} playing {scoreInfo.Beatmap.GetDisplayTitle()}";
+        public static string GetDisplayTitle(this IScoreInfo scoreInfo) => $"{scoreInfo.User.Username} playing {scoreInfo.Beatmap.GetDisplayTitle()} ({scoreInfo.Date.LocalDateTime:yyyy-MM-dd})";
```

Comment:
I don't particularly have anything against the spirit of this change but the placement makes me extremely uneasy. `GetDisplayTitle()` was intended as a general-purpose function one can call when wanting a string representation of a model. The choice of an export filename is only one use of this method. Not to mention that `ScoreInfo.ToString()` delegates to this method, too, so every single time that a score is formatted as a string or even string-interpolated will now receive a date.

I'd much rather this was scoped down to the score exporter specifically rather than shotgunned into this method.